include/ck/host_utility/kernel_launch.hpp

```diff
@@ -156,3 +156,82 @@ float launch_and_time_kernel_with_preprocess(const StreamConfig& stream_config,
     return 0;
 #endif
 }
+
+// return time that does not include preprocess's time
+template <typename... Args, typename F, typename PreProcessFunc>
+float launch_and_time_kernel_without_preprocess(const StreamConfig& stream_config,
+                                                PreProcessFunc preprocess,
+                                                F kernel,
+                                                dim3 grid_dim,
+                                                dim3 block_dim,
+                                                std::size_t lds_byte,
+                                                Args... args)
+{
+#if CK_TIME_KERNEL
+    if(stream_config.time_kernel_)
```

Comment:
If we want to use flush cache, we always want to time the kernel, is this if condition could be removed?

Reply:
I hope this is a universal function, just like launch_and_time_kernel_with_process, which is used to calculate the time of preprocess as well

Reply:
done, add TimePreprocess parameter to launch_and_time_kernel_with_preprocess